CardGame/src/components/header/Header.tsx

```diff
@@ -0,0 +1,59 @@
+import { styled } from "styled-components";
+import Button from "@/components/button/Button";
+
+interface HeaderInterface {
+  score: number;
+  numPairs: number;
+  onClickReset: () => void;
+}
+
+const Header = ({ score, numPairs, onClickReset }: HeaderInterface) => {
+  return (
+    <HearderStyled>
+      <TitleBoxStyled>
+        <TitleStyled>마루 카드 맞추기</TitleStyled>
+        <ScoreStyled>
+          {score} / {numPairs}
+        </ScoreStyled>
+      </TitleBoxStyled>
+      <BtnWrapper>
+        <Button
+          text="Reset"
+          color="var(--point-color)"
+          onClick={onClickReset}
+        />
+      </BtnWrapper>
+    </HearderStyled>
+  );
+};
+
+const HearderStyled = styled.div`
```

Comment:
HeaderStyled !

Reply:
웹 접근성 고려해서 시멘틱한 태그 사용하는 건 어떨까요~
```suggestion
const HearderStyled = styled.header`
```